news/2022-08-30-changes-to-osu-sr-and-pp.md

```diff
@@ -0,0 +1,97 @@
+
+#### Repeated angles
+
+Maps with Flashlight applied become significantly easier if the successive object placement is predictable. One of the major ways this can manifest is through the use of repeated angles, which [this change](https://github.com/ppy/osu/pull/19716) aims to nerf.
+
+Throughout the development process for the Flashlight changes, a number of grid-style maps such as [DJPop](https://osu.ppy.sh/users/2363)'s [beatMARIO - Night of Knights [SOLO]](https://osu.ppy.sh/beatmapsets/15920#osu/58063) became increasingly concerning as their values rose. By nerfing repeat angles, grid-style maps such as this one become nerfed to a more suitable value.
+
+#### Slider bonus
+
+Fast and long sliders introduce additional difficulty to Flashlight play, so a slider bonus is [now applied](https://github.com/ppy/osu/pull/15666)! The length of a slider wasn't considered at all up until now.
+
+The bonus only kicks in once a slider is fast enough since slow sliders are easily readable, and the bonus also has a distance factor to account for longer sliders.
+
+#### Bug fixes
+
+- [**Remove decay factor.**](https://github.com/ppy/osu/pull/15728) The skill was accidentally double-dipping by having two methods of decaying objects operating simultaneously. The decay factor was removed, buffing mostly stream maps in the process.
+- [**Fix cumulative strain time calculation.**](https://github.com/ppy/osu/pull/15867) One of the factors which the Flashlight skill uses to calculate memorisation difficulty is the gap in time between a given object and one of its previous objects. The calculation was incorrect and is now fixed.
+- [**Remove difficulty spike nerf.**](https://github.com/ppy/osu/pull/18791) This nerf was a leftover from the aim and speed skills and it didn't make much sense with respect to non-mechanical difficulty skills such as Flashlight so it was removed.
+
+### Minor changes
+
+Alongside all those big changes above, there are some changes on the smaller side of things:
+
+- [**The speed skill now measures distance correctly.**](https://github.com/ppy/osu/pull/15758) [GoldenMine](https://osu.ppy.sh/users/8058206) spotted that distance was calculated improperly since the last change, causing aim from sliders to be potentially overemphasised. This has now been corrected.
+- [**Miss count estimate is now smooth.**](https://github.com/ppy/osu/pull/16331) Previously, the miss count estimate could jump from 1 miss to 2 misses within a combo, but this transition is done more smoothly now that the estimated miss count is calculated as a decimal value, thanks to [StanR](https://osu.ppy.sh/users/7217455).
+- [**1x100 scores no longer break miss count estimate.**](https://github.com/ppy/osu/pull/19804) [StanR](https://osu.ppy.sh/users/7217455) rectified an issue where a single 100 on a score would potentially count as more than one miss.
+
+---
+
+We hope you enjoy the changes and blah blah I can't really write outros so I hope my wiki friends help
```

Comment:
maybe take inspiration from the other two related news posts 

```suggestion
Hopefully these changes will help you measure your pp more accurately. Join the [osu!dev Discord server](https://discord.gg/ppy) and maybe even the dedicated [Performance Points Discord server](https://discord.gg/aqPCnXu), where you can leave valuable feedback to the people working hard to make pp measurement algorithms more accurate.
```

(ahem excuse me)

```suggestion
If you're curious about the development of performance points, have a look at the `#difficulty-osu` channel in the [osu!dev Discord server](https://discord.gg/ppy). If you're *really* curious about the under-the-hood details, then we're happy to tell you that there is also a dedicated [Performance Points Discord server](https://discord.gg/aqPCnXu), where much of the development and discussion is organised.
```

and well idk what the order in these news posts will be, but an idea could be to add "there seems to be even more news about sr and pp coming very soon..." or smth